config/initializers/money.rb

```diff
@@ -79,7 +79,23 @@
     iso_numeric:            "643",
     smallest_denomination:  1
   }
-
+  
+  # West african francs
+  config.register_currency = {
+    priority:               1,
+    iso_code:               "XOF",
+    name:                   "West African Cfa Franc",
+    symbol:                 "CFA",
+    alternate_symbols:      ["Fr"],
+    subunit:                "centime",
```

Comment:
Should this be Centime? Like in here: https://github.com/RubyMoney/money/blob/master/config/currency_iso.json#L2478